beetsplug/replaygain.py

```diff
@@ -1381,17 +1433,30 @@ def commands(self):
         def func(lib, opts, args):
             write = ui.should_write(opts.write)
             force = opts.force
+            jobs = opts.jobs
```

Comment:
Instead of defaulting to off when jobs = 0, let's use our `cpu_count` utility:
https://github.com/beetbox/beets/blob/c1937e1b01d7130a27cdb3a5f589bb4f1bab9ce6/beets/util/__init__.py#L760

Reply:
Should we do something like default = math.ceil(cpu_count() / 4) to keep CPU usage reasonable by default for large queries?

Reply:
And maybe have it default to no parallelism if configured with `threaded: no`?

Reply:
Maybe the best policy would be to match the behavior of the `convert` plugin:
https://github.com/beetbox/beets/blob/master/beetsplug/convert.py#L119

The option is named `threads` there, and it does default to the CPU count. I think it's probably OK to default to that? If people want to run in the background without disruption, it's easy to turn down the count…